workspace.go

```diff
@@ -379,6 +385,15 @@ type WorkspaceCreateOptions struct {
 	// exist, it is created and added to the workspace.
 	Tags []*Tag `jsonapi:"relation,tags,omitempty"`
 
+	// Struct containing boolean values that indicate whether the resolved value of
+	// a setting should be decided by something other than the workspace (i.e. a related
+	// organization or project).
+	//
+	// For example: When setting the execution-mode of a workspace to  "default", the
+	// `setting-overwrites.execution-mode` field should be set to true, because the execution-mode
```

Comment:
This is backwards. If the value is `true`, that means that the workspace is overwriting the value.

Reply:
... woops. Good catch!

Reply:
I think you might be missing the `to "default"` part. Either that, or all the testing I've done for the API is completely wrong